packit/cli/prepare_sources.py

```diff
@@ -90,6 +90,13 @@ def load_job_config(job_config):
     type=click.STRING,
     help="Specifies target branch which PR should be merged into.",
 )
+@click.option(
+    "--create-symlinks/--no-create-symlinks",
```

Comment:
Can we please make it singular since only a single archive will be processed?

Reply:
resolved via chat: I was considering we may create symlinks in multiple places during running `prepare-sources` in future and in Copr environment, we will want to not create symlinks anywhere. I will add a comment about this to the code